### DISCUSSION TITLE
Maximum number of records in mass edition - limit

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#16980] [How to]
Hi all

I increased "Maximum number of records in mass edition" in Config.Templates.php to 50000 but still I cannot edit this number of records.
The max in System – Configuration field: "Maximum number of records in mass edition" I can enter is 5000
![image](https://user-images.githubusercontent.com/35586696/227704407-29534255-40b7-4497-81a8-dcfbad270e0e.png)

How to increase this value?

**Comment by rskrzypczak** [comment]
The maximum allowed value is 5000, if you need more then you have to change it manually directly in `config/Main.php`.
